README.md

```diff
@@ -1,25 +1,23 @@
-# FirebaseUI for iOS — UI Bindings for Firebase
+# FirebaseUI for iOS — UI Bindings for Firebase [![Build Status](https://travis-ci.org/firebase/FirebaseUI-iOS.svg?branch=master)](https://travis-ci.org/firebase/FirebaseUI-iOS)
 
-[![Build Status](https://travis-ci.org/firebase/FirebaseUI-iOS.svg?branch=master)](https://travis-ci.org/firebase/FirebaseUI-iOS)
-
-FirebaseUI is an open-source library for iOS that allows you to quickly connect common UI elements to the [Firebase](https://firebase.google.com?utm_campaign=Firebase_update_awareness_general_en_05-18-16_&utm_source=?utm_campaign=Firebase_featureoverview_awareness_analytics_en_05-18-16_&utm_source=?utm_campaign=Firebase_announcement_awareness_general_en_05-18-16_&utm_source=Firebase&utm_medium=ads&utm_medium=?utm_campaign=Firebase_announcement_awareness_general_en_05-18-16_&utm_source=Firebase&utm_medium=ads&utm_medium=?utm_campaign=Firebase_update_awareness_auth_en_05-18-16_&utm_source=?utm_campaign=Firebase_announcement_awareness_general_en_05-18-16_&utm_source=Firebase&utm_medium=ads&utm_medium=?utm_campaign=Firebase_featureoverview_awareness_analytics_en_05-18-16_&utm_source=?utm_campaign=Firebase_announcement_awareness_general_en_05-18-16_&utm_source=Firebase&utm_medium=ads&utm_medium=?utm_campaign=Firebase_announcement_awareness_general_en_05-18-16_&utm_source=Firebase&utm_medium=ads) database for data storage, allowing views to be updated in realtime as they change, and providing simple interfaces for common tasks like displaying lists or collections of items.
+FirebaseUI is an open-source library for iOS that allows you to quickly connect common UI elements to the [Firebase](https://firebase.google.com) database for data storage, allowing views to be updated in realtime as they change, and providing simple interfaces for common tasks like displaying lists or collections of items.
```

Comment:
Sweet mother of UTM campaigns. Let's keep a `utm_source=FirebaseUI-iOS` at the minimum... (but yeah, who added the other ones?)